README.md

```diff
@@ -2,29 +2,32 @@
 lab manager for plate maps and sequence flows
 
 # Install
-Labman relies on the Qiita database. You will need first to install Qiita in
-a different environment (Qiita is Python 2 only, while labman is Python 3) and
-create the Qiita database using the [Qiita installation instructions](https://github.com/biocore/qiita/blob/master/INSTALL.md).  The instructions
+LabControl relies on the Qiita database and webserver. You will need first to 
+install Qiita in a different environment than LabControl (Qiita is Python 2 
```

Comment:
Just an FYI, Labcontrol is testing in CI against the dev version of Qiita, which now requires Python 3.6. The next release of Qiita will be 3.6-based, so the information within the parentheses will become obsolete. However, they will continue to require separate Conda environments due to packaging dependencies.

Reply:
@charles-cowart Good to know! I have removed the reference to python 2 in the LabControl readme.  Note that the LabControl readme refers people to the Qiita installation instructions, so these definitely MUST be updated before the next Qiita release to reflect the python version change--if they haven't already been!

Reply:
Will do!